src/User/Domain/User.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+namespace App\User\Domain;
+
+use Doctrine\ORM\Mapping as ORM;
+
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="users")
+ */
+class User
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(name="id", type="integer", nullable=false)
+     */
+    public $id;
+    /**
+     * @ORM\Column(name="email", type="string", length=100, nullable=false)
+     */
+    public $email;
+    /**
+     * @ORM\Column(name="first_name", type="string", length=100, nullable=false)
+     */
+    public $firstName;
```

Comment:
Are you sure that you will first name and last name from google? I think the user can be without  the first and the last name  requirements